src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -31,25 +32,84 @@ const styles = {
   }),
 };
 
+const getValueRecursively = (options, value) => {
+  if (!value) {
+    return null;
+  }
+
+  const option = options.find(opt => opt.value === value);
+  if (option) {
+    return option;
+  }
+
+  // Search for the option recursively
```

Comment:
is this really needed? we only use one level of option groups, no? a generic recursive function has some potential/risk to recurse infinitely, I'd prefer to have a more simple approach tailored to how we use react-select here

Reply:
Mhh, fair enough. I've replaced it with a more tailored function